mep/pages/models.py

```diff
@@ -110,14 +118,76 @@ class LandingPage(RdfPageMixin, Page):
     ]
 
 
+class RoutableLandingPage(LandingPage):
```

Comment:
I agree this isn't a great name, it's too generic. Can we reference the fact that it's more of a feed, or by date? NewestFirst ? FeedLandingPage ? BloggishLandingPage ? 

I expect that eventually it will be paginated and the routing will need to be updated to handle that too.

Reply:
I also wonder if the fact that you're extending `LandingPage` is making this more complicated (possibly contributing to the issues in the testing and fixtures). It might be simpler (at least in terms of database tables) to make an abstract base class (`BaseLandingPage`, maybe) that both landing pages extend from.